src/test/java/org/opentripplanner/middleware/triptracker/ManageLegTraversalTest.java

```diff
@@ -216,6 +221,7 @@ private static Stream<Arguments> createTurnByTurnTrace() {
         Coordinates originCoords = new Coordinates(adairAvenueToMonroeDriveLeg.from);
         Coordinates destinationCoords = new Coordinates(adairAvenueToMonroeDriveLeg.to);
         Coordinates adairAvenueNortheastCoords = new Coordinates(adairAvenueNortheastStep);
+        Coordinates midtownWalkCoords = new Coordinates(33.784372, -84.381410); //33.784382, -84.381743);
```

Comment:
Remove comment.

Reply:
Thanks, removed in https://github.com/ibi-group/otp-middleware/pull/300/commits/d359992e8ef3b209240be2e6022348f57b7598c0